subprojects/core/src/integTest/groovy/org/gradle/api/internal/plugins/ApplyPluginBuildOperationIntegrationTest.groovy

```diff
@@ -67,63 +59,139 @@ class ApplyPluginBuildOperationIntegrationTest extends AbstractIntegrationSpec {
             null,
             null,
         ]
+    }
```

Comment:
I'm surprised by this test results. Didn't we add an id to all plugins in the Gradle codebase a while ago?